device/vxlan/unicast.yaml

```diff
@@ -1,45 +1,74 @@
 components:
   schemas:
-    Vxlan.Unicast:
+    Vxlan.V4Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V4Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.V6Tunnel.DestinationIPMode.Unicast:
+      properties:
+        vteps:
+          description: |-
+            List of VTEPs for member VNI(VXLAN Network Identifier)
+          type: array
+          items:
+            $ref: '#/components/schemas/Vxlan.V6Tunnel.DestinationIPMode.Unicast.Vtep'
+
+    Vxlan.Tunnel.DestinationIPMode.Unicast.VtepBase:
       description: >-
-        Configuration of Unicast parameters for VxLAN.
+        DestinationIPMode Unicast VTEP Base
       type: object
       properties:
-        remote_vtep_address:
-          description: >-
-            Remote VXLAN Tunnel End Point address
-          $ref: '#/components/schemas/Remote.Vtep.Address'
-        suppress_arp:
-          description: >-
-            Supress ARP 
-          type: boolean
-          default: false
-        remote_vm_mac:
-          description: >-
-            Remote VM MAC addresses
+        arp_suppression_cache:
+          description: >- 
+            Arp Suppression Cache
           type: array
           items:
-            type: string
-            format: mac
+            $ref : '#/components/schemas/Vxlan.Tunnel.DestinationIPMode.Unicast.ArpSuppressionCache'
+
+    Vxlan.Tunnel.DestinationIPMode.Unicast.ArpSuppressionCache:
+      description: >-
+         ARP suppression reduces ARP broadcast traffic by using ARP Suppression Cache.
```

Comment:
VM arps are supressed by resolving against the static locally configured db.If arp entry is not found in the local static db,arp requests are send

Reply:
arp suppression cache is per remote vtep

Reply:
in multicasy yes.always multicast to all vteps,vni number determines the multicast address.yes static db can be there for multicast,I dont see a resson why it shoul'nt be.ixn has done this additional stuff for unicast only